src/tests/whitespace_test

```diff
@@ -35,3 +35,15 @@ fi
             exit found
         }
     "
+
+declare found_file=0
+git ls-files | \
```

Comment:
Can you use the `--exclude` option to `git ls-files`?

Like this:

    git ls-files --exclude="src/config/testconfigs/noparse.api.conf" \
                 --exclude="src/tests/cmocka/p11_nssdb/.*db"